app/src/main/res/layout/profile_progress_header.xml

```diff
@@ -12,36 +12,34 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
-
-    <FrameLayout
-      android:id="@+id/profile_frame_layout"
-      android:layout_width="96dp"
-      android:layout_height="96dp"
-      android:layout_marginTop="32dp"
+    android:layout_height="wrap_content"
+    android:clipToPadding="false"
+    android:paddingTop="32dp">
+
+    <com.jackandphantom.circularimageview.CircleImage
```

Comment:
In HDPI device this is looking like this which looks odd.
Tested on Nexus S.
<img width="199" alt="Screenshot 2020-04-14 at 11 55 27 AM" src="https://user-images.githubusercontent.com/9396084/79192734-e69bc480-7e46-11ea-858d-144f0a64a9cf.png">

Reply:
Issue is created for the same.
https://github.com/oppia/oppia-android/issues/980